### PR TITLE
Add capy-power map

### LINKED ISSUE #998: Create image map on the homepage capybara
### What's your idea?

As suggested by @EvanCPSC, create an interactive feature on the homepage image of the capybara so that when the mouse hovers over the image, the team icons get larger.

image

Reference: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/map

### PR DESCRIPTION
**Pull request by EthanThatOneKid**
Resolves #998.

### Screenshot

58133172-CB98-47D8-8570-F82ED599AB8B

### CODE REVIEW

File: src/routes/(site)/hero.svelte [hunk 1]
```diff
@@ -14,7 +14,70 @@
...
+      <area
+        alt="Game Dev Team"
+        title="Game Dev Team"
+        href="/teams#gamedev"
+        coords="90,60 50,100 90,140 130,100"
+        shape="polygon"
+      />
+    </map>
+
+    <img
```
Comment (EthanThatOneKid) [inline src/routes/(site)/hero.svelte:76]: Caught by @karnikaavelumani:

image

Comment (EthanThatOneKid) [reply src/routes/(site)/hero.svelte:76]: Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/1011/commits/3fb1fa5082f802bced2515922380363836d466f1>.

**Review by k4rni — APPROVED** [review]
This is incredible! I checked all the icons and they refer back to the correct location in the /teams page. Often times I see many websites reference back to the wrong location, which is a personal pet peeve. Thank you for taking up this issue.

**Review by tomasohCHOM — COMMENTED** [review]
LGTM!

**Pull request merged**